### PR TITLE
Fix link to mysql 4byte docs

### PR DESCRIPTION
**Pull request by nickvergessen**
Signed-off-by: Joas Schilling <coding@schilljs.com>

### CODE REVIEW

**Review by jospoortvliet — APPROVED** [review]

File: admin_manual/maintenance/backup.rst [hunk 1]
```diff
@@ -48,7 +48,7 @@ database engine. To backup MySQL/MariaDB::
 
     mysqldump --single-transaction -h [server] -u [username] -p[password] [db_name] > nextcloud-sqlbkp_`date +"%Y%m%d"`.bak
 
-If you use `4 byte support <https://docs.nextcloud.com/server/latest/go.php?to=mysql_4byte_support`_ in MySQL/MariaDB (needed for emoji), you will need to add ``--default-character-set=utf8mb4`` like this::
+If you use enabled MySQL/MariaDB 4-byte support (:doc:`../configuration_database/mysql_4byte_support`, needed for emoji), you will need to add ``--default-character-set=utf8mb4`` like this::
```
Comment (MorrisJobke) [inline admin_manual/maintenance/backup.rst:51]: If you ~use~ enabled

Comment (nickvergessen) [reply admin_manual/maintenance/backup.rst:51]: arg

**Pull request merged**